plugins/themes/default/DefaultThemePlugin.inc.php

```diff
@@ -17,17 +17,10 @@
 
 class DefaultThemePlugin extends ThemePlugin {
 	/**
-	 * Constructor
-	 */
-	function DefaultThemePlugin() {
```

Comment:
Generally I've been including them. Might be worth reviewing at some point, both to make inclusion/exclusion consistent, and probably to change over to newer PHP syntax, but ... later. I think these used to be strictly necessary, way back when in PHP4 days, if you wanted parent class constructors to be called.